lib/github/api.nit

```diff
@@ -138,11 +145,11 @@ class GithubAPI
 
 	# Load the json object from Github.
 	# See `GithubEntity::load_from_github`.
-	protected fun load_from_github(key: String): JsonObject do
+	protected fun load_from_github(key: String): nullable GithubEntity do
 		message(1, "Get {key} (github)")
 		var res = get(key)
-		if was_error then return new JsonObject
-		return res.as(JsonObject)
+		if was_error then return null
+		return deserialize(res.as(JsonObject).to_json).as(nullable GithubEntity)
```

Comment:
* you have an object `res`
* you cast is as a `JsonObject`
* you serialize it as a JSON string
* you deserialize it as an Object

This seems quite convoluted to me (but I'm not an expert)